### ISSUE TITLE
Download page update

### ISSUE THREAD

**Issue by rgrp** [cybergreen-net/stats-new#5]
* [x] Rename (?) to "Download and API"
* [ ] Bulk download (from s3 bits.cybergreen.net) of entire entire entries dataset and other dataset
* [x] Documentation on API

## What do we have on S3

* count.csv
* count_by_country.csv
* count_by_risk.csv

## Reference data

**Reference data** - on github i think

We link to github raw urls by default. In a perfect world (post sep) we have data packages in github and we can auto display them :-)

* risk list - this includes the numeric ids now. This may be waiting on https://github.com/cybergreen-net/pm/issues/6
* asn list
* country list

**Comment by aaronkaplan** [comment]
by "entire dataset" I hope you do **not** mean the disaggregated data? Only aggregated

**Comment by rgrp** [comment]
@aaronkaplan yes, everything here refers to aggregated data. There is no disaggregated data in the frontend or available from it.

**Comment by aaronkaplan** [comment]
Perfect, just checking. As said: +1 from me.

**Comment by rgrp** [comment]
FIXED.

**Comment by rgrp** [comment]
I am reopening as I realize now we are missing the link to the entire entries dataset.

@zelima please could you

* Add a link in the CSV list section to the entire count data called: `Full count data (by ASN, Country, and week)`
* Delete link to Country rank data as this should not be there, at least for now (it currently links to fixtures which cannot be right)
* I would entirely eliminate the JSON section on the right as that is superseded by the API (and the JSON data is not bulk)
